legend-engine-authentication-protocol/src/main/java/org/finos/legend/engine/protocol/pure/v1/model/packageableElement/auth/credential/PlainTextCredential.java

```diff
@@ -0,0 +1,17 @@
+package org.finos.legend.engine.protocol.pure.v1.model.packageableElement.auth.credential;
```

Comment:
package seems weird, why these are under packageable elements? Are we adding these as packageable elements?

Reply:
Its not a packageable element, moved it to legend-engine-shared-core